src/main/java/org/gridsuite/filter/server/dto/LineFilter.java

```diff
@@ -36,34 +37,32 @@ public EquipmentType getEquipmentType() {
     String substationName2;
 
     @Schema(description = "Countries1")
-    private SortedSet<String> countries1;
+    private Set<String> countries1;
 
     @Schema(description = "Countries2")
-    private SortedSet<String> countries2;
+    private Set<String> countries2;
+
+    @Schema(description = "Free properties 1")
+    private Map<String, Set<String>> freeProperties1;
+
+    @Schema(description = "Free properties 2")
+    private Map<String, Set<String>> freeProperties2;
 
     @Schema(description = "Nominal voltage 1")
     private NumericalFilter nominalVoltage1;
 
     @Schema(description = "Nominal voltage 2")
     private NumericalFilter nominalVoltage2;
 
-    public LineFilter(String equipmentID, String equipmentName, String substationName1, String substationName2, SortedSet<String> countries1, SortedSet<String> countries2, NumericalFilter nominalVoltage1, NumericalFilter nominalVoltage2) {
```

Comment:
why removing the constructor ?

Reply:
Because the builder does the job better.